packages/atomic/src/utils/utils.ts

```diff
@@ -186,6 +186,11 @@ export function isInDocument(element: Node) {
   return false;
 }
 
+export function isVisible(element: Node) {
+  const style = window.getComputedStyle(element as HTMLElement);
+  return style.display !== 'none';
+}
```

Comment:
It's not just checking for the `atomic-hidden` class that I'd like to avoid, I'd like to avoid anything that isn't explicitly provided by the facets.

My worry is that future changes may hide the facet in a different way or an implementer may add their own styling which would affect this function.

Reply:
Okay, I am working on the isVisible Atomic method and for now, I have to add it to all of these facets.

<img width="888" alt="image" src="https://github.com/coveo/ui-kit/assets/132079077/166b197c-00c8-42f1-8baf-53bed502b7cd">

Also, since public methods have to be async in stencil, it gets really hard to use them inside of stencil components. I tried multiple ways and nothing is working. 😢 

I think adding a isVisible to every component would almost be harder to remember to change then having two places where we check if the display is none :/